src/StreamService/appsettings.Development.json

```diff
@@ -0,0 +1,21 @@
+{
+  "Serilog": {
+  },
+  "Kestrel": {
+    "EndPoints": {
+      "Http": {
+        "Url": "http://[:]:9838"
+      }
+    }
+  },
+  "ConnectionStrings": {
+  },
+  "RabbitMQ": {
+    "Host": "localhost",
+    "Username": "guest",
+    "Password": "guest"
+  },
```

Comment:
Add cacheLive for memory cache

Reply:
appsettings.Production contains it, should it be in Development?